frontend/app/routes/$lang+/_protected+/applications+/$id.tsx

```diff
@@ -0,0 +1,91 @@
+import { toLocaleDateString } from '~/utils/date-utils';
+import { featureEnabled } from '~/utils/env.server';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { getFixedT, getLocale } from '~/utils/locale-utils.server';
+import { IdToken } from '~/utils/raoidc-utils.server';
+import { RouteHandleData } from '~/utils/route-utils';
+
+export const handle = {
+  breadcrumbs: [{ labelI18nKey: 'applications:index.page-title' }],
+  i18nNamespaces: getTypedI18nNamespaces('applications', 'gcweb'),
+  pageIdentifier: pageIds.protected.applications.view,
+  pageTitleI18nKey: 'applications:index.page-title',
+} as const satisfies RouteHandleData;
+
+export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
+  featureEnabled('view-applications');
+
+  const auditService = getAuditService();
+  const instrumentationService = getInstrumentationService();
+  const raoidcService = await getRaoidcService();
+  await raoidcService.handleSessionValidation(request, session);
+
+  if (!params.id) {
+    instrumentationService.countHttpStatus('application.view', 400);
+    throw new Response(null, { status: 400 });
+  }
+
+  //prevent users from entering any ID in the URL and seeing other users' applications
+  const applications: { id: string | undefined; submittedOn: string | undefined; status: string | undefined; confirmationCode: string | undefined }[] | undefined = session.get('applications');
+  const viewApplication = applications?.find((applications) => applications.id === params.id);
+
+  if (!viewApplication) {
+    instrumentationService.countHttpStatus('application.view', 404);
+    throw new Response(null, { status: 404 });
+  }
+  const locale = await getLocale(request);
+  const submittedParsedFormat = viewApplication.submittedOn ? toLocaleDateString(new Date(viewApplication.submittedOn), locale) : undefined;
+
+  const t = await getFixedT(request, handle.i18nNamespaces);
+  const meta = { title: t('gcweb:meta.title.template', { title: t('applications:index.page-title') }) };
+
+  const idToken: IdToken = session.get('idToken');
+  auditService.audit('page-view.applications', { userId: idToken.sub });
+  instrumentationService.countHttpStatus('applications.view', 200);
+
+  return json({ viewApplication, submittedParsedFormat, meta });
+}
+
+export default function ViewApplication() {
+  const params = useParams();
+  const { t } = useTranslation(handle.i18nNamespaces);
+  const { viewApplication, submittedParsedFormat } = useLoaderData<typeof loader>();
+  // TODO: Update with field mappings from Benefit Application from AB-3382
+  return (
+    <>
+      <div className="max-w-prose">
+        <div className="space-y-10">
+          <div className="space-y-6">
+            <h2 className="text-2xl font-semibold">{t('applications:index.page-title')}</h2>
+            <h2 className="text-2xl font-semibold">{t('applications:view-application.page-sub-title')}</h2>
```

Comment:
Fixed: 1e363b7515ed32ceb9084798bea09b9f427b6b0b.

Reply:
Most of the points above are fixed. There is still an extra `<h2>` that should be removed IMO, otherwise we have a repeating title:

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/be4a8916-74d6-4c3d-9262-2a2d2b255490)